samples/bootloader/README.rst

```diff
@@ -85,12 +85,11 @@ The bootloader uses the :ref:`doc_bl_storage` library to access provisioned data
 Requirements
 ************
 
-* One of the following development boards:
+The sample supports the following development kits:
 
-  * |nRF9160DK|
-  * |nRF52840DK|
-  * |nRF52DK|
-  * |nRF51DK|
+.. include:: /includes/boardname_tables/sample_boardnames.rst
+   :start-after: misc_samples_set1_start
+   :end-before: misc_samples_set1_end
```

Comment:
This set seems to miss nrf51dk?

Reply:
We don't officially support nRF51 DK in the NCS.